### DISCUSSION TITLE
How to get embedding vectors after training?

### DISCUSSION THREAD

**Discussion by EdwinSong** [shenweichen/DeepCTR-Torch#222] [Q&A]
After training, we need get out the embedding vectors, look the code inside and out, I guess I could use below method, please confirm whether it is right or is there any right approach?
-- 
list_vector = model.embedding_dict['<field name>'].weight.data
--

As we are using label encoding, also want to confirm the list_vector match the sequence of index of lbe.classes_, for example, list_vector[0] is the embedding vector of lbe.classes_ first key(label), after LabelEncoder fit_transform()

Thanks a lot!
